### PR TITLE
Standardise Makefile

### PR DESCRIPTION
**Pull request by sedroche**
## Motivation
As a general rule we should be able to move across projects and use the same set of commands to build and test them.

## What
Applied the proposal outlined in https://issues.jboss.org/browse/INTLY-118

## Verification Steps

1. Run the commands

## Checklist:

- [x] Code has been tested locally by PR requester
- [ ] Changes have been successfully verified by another team member

### CODE REVIEW

**Comment by sedroche** [comment]
@odra I added `code/check` to the travis test job so I had to add a commit to run `code/fix`

File: Makefile [hunk 1]
```diff
@@ -23,67 +23,86 @@ travis/setup:
...
-image/docker-build: res/copy code/compile-for-docker
-	docker build -t ${REG}/${ORG}/${IMAGE}:${TAG} -f build/Dockerfile .
+.PHONY: test/e2e/cluster
+test/e2e/cluster: image/build/with/tests image/push
+	oc apply -f deploy/test-pod.yaml -n ${NS}
+	${SHELL} ./scripts/stream-pod ${TEST_POD_NAME} ${NS}
 
-image/build-with-tests: res/copy
+.PHONY: image/build/with/tests
+image/build/with/tests:
```
Comment (odra) [inline Makefile]: I think `image/build/tests` looks better

**Comment by odra** [comment]
@sedroche looks ok, just one comment.

File: Makefile (hunk 1 above)
Comment (sedroche) [reply Makefile]: :+1:

**Review by odra — APPROVED** [review]

**Comment by odra** [comment]
feel free to merge once travis tests are green.

**Review by odra — APPROVED** [review]

**Pull request merged**